CONTRIBUTING.md

```diff
@@ -27,6 +27,44 @@ Pull requests are the best way to propose changes to the codebase (we use [Githu
 7. Make sure your code lints.
 8. Issue that pull request!
 
+## Setup Debug Environment
+
+Install requirements:
+
+```shell
+pip install -r requirements.txt
+```
+
+Use jc in development mode (changes are direct available):
+
+```shell
+python3 setup.py develop
+```
+
+Uninstall local development:
+
+```shell
+python3 setup.py develop --uninstall
+```
+
+Use `--debug` to see error messages (double to see more):
+
+```shell
+echo 'abc' | jc --parser-with-error --debug
+```
+
+## Install `jc` with `pip` from source
+
+Please note: Development mode is preferred for observing instant changes.
+
+```shell
+pip install .
```

Comment:
Should we do something like:
```
pip3 install --upgrade --user -e .
```
This is what `install.sh` does. It basically puts it in developer mode for instant changes and only for the current user.

Reply:
ah - I was not aware of this `-e` option :)

I would apply your suggestion to the text

Reply:
the pip install not works on my mac :(
I have only success with: `python3 setup.py develop`

```shell
$ jc --version
jc version:  1.24.1
python interpreter version:  3.11.6
python path:  /opt/homebrew/opt/python@3.11/bin/python3.11

https://github.com/kellyjonbrazil/jc
© 2019-2024 Kelly Brazil

$ ./uninstall.sh 
Found existing installation: jc 1.24.1
Uninstalling jc-1.24.1:
  Would remove:
    /opt/homebrew/bin/jc
    /opt/homebrew/lib/python3.11/site-packages/jc.egg-link
Proceed (Y/n)? y
  Successfully uninstalled jc-1.24.1
$ jc --version 
zsh: command not found: jc
$ ./install.sh 
Obtaining file:///Users/muescha/Work/github.com/muescha/jc
  Preparing metadata (setup.py) ... done
Requirement already satisfied: ruamel.yaml>=0.15.0 in /opt/homebrew/lib/python3.11/site-packages (from jc==1.24.1) (0.18.5)
Requirement already satisfied: xmltodict>=0.12.0 in /opt/homebrew/lib/python3.11/site-packages (from jc==1.24.1) (0.13.0)
Requirement already satisfied: Pygments>=2.3.0 in /opt/homebrew/lib/python3.11/site-packages (from jc==1.24.1) (2.17.2)
Requirement already satisfied: ruamel.yaml.clib>=0.2.7 in /opt/homebrew/lib/python3.11/site-packages (from ruamel.yaml>=0.15.0->jc==1.24.1) (0.2.8)
Installing collected packages: jc
  Running setup.py develop for jc
Successfully installed jc-1.24.1
$ jc --version
zsh: command not found: jc

```

Reply:
tested some options. Without `--user` I have success:

```shell
pip3 install --upgrade -e .
```